### PR TITLE
Batch carol app review. 

### PR DESCRIPTION
**Pull request by rafarui**
Too many things in the other PR. I created this one just to validate the batch app

### CODE REVIEW

File: batch_carolapp/batch_carolapp/manifest.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    {
+      "dockerName": "batch_carolapp-batch_carolapp_algorithm",
+      "dockerTag": "0.1.0",
+      "gitBranch": "app_review",
+      "gitPath": "/batch_carolapp",
+      "instanceType": "c1.small",
+      "gitDockerfileName": "Dockerfile",
+      "gitRepoUrl": "https://github.com/totvslabs/carolapp-samples"
+    }
+  ]
```
Comment (gzamboni) [inline batch_carolapp/batch_carolapp/manifest.json:39]: That is this part? this is new?

**Comment by gzamboni** [comment]
Those new things gonna be ported to cookiecutter projects?

File: batch_carolapp/batch_carolapp/manifest.json (hunk 1 above)
Comment (rafarui) [reply batch_carolapp/batch_carolapp/manifest.json:39]: yeap.. this is new..
it is the docker build parameters.

Comment (rafarui) [reply batch_carolapp/batch_carolapp/manifest.json:39]: then carol know what to build

**Comment by rafarui** [comment]
> Those new things gonna be ported to cookiecutter projects?

I think they should

**Pull request merged**